### PR TITLE
Updated band details for Sentinel5P with reference to openEO hub collections

### PR DESCRIPTION
**Pull request by Pratichhya**
This is partial task accomplished to Issue #174 dedicated to Sentinel5P only.

### CODE REVIEW

**Comment by Pratichhya** [comment]
Seems to be display issue only in Sentinel 5P in the publish branch. Not sure for the reason working on it in #231

**Comment by jdries** [comment]
Hi @Pratichhya , when I look in the netCDF files, the band names are different, for instance: 
ozone_total_vertical_column instead of O3_column_number_density
Also the readm's seem to refer to variable names from the netCDF, which makes sense. Did we get these names from somewhere else?

**Comment by Pratichhya** [comment]
> Hi @Pratichhya , when I look in the netCDF files, the band names are different, for instance: ozone_total_vertical_column instead of O3_column_number_density Also the readm's seem to refer to variable names from the netCDF, which makes sense. Did we get these names from somewhere else?

Ah, no. I extracted the metadata data from either openEO hub or Planetary. I wasn't sure if I have to fetch each collection in dataspace and then extract from it or there is other way around.

**Comment by jdries** [comment]
There's not really a fixed approach, but we try to stick as close as possible to what ESA provides. 
In this case, I inspected available variables in the netcdf file like shown below. 
(This is quite a complex case.)

```

gdalinfo /eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc
Driver: netCDF/Network Common Data Format
Files: /eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc
Size is 512, 512
Metadata:
  NC_GLOBAL#algorithm_version=UPAS-O3-DOAS_CAL-5.1.0
  NC_GLOBAL#build_date=22-06-2022T12:42:53Z
  NC_GLOBAL#cdm_data_type=Swath
  NC_GLOBAL#cloud_mode=cal
  NC_GLOBAL#Conventions=CF-1.7
  NC_GLOBAL#cpp_compiler_flags=-std=gnu++11 -O2 -fopenmp 
  NC_GLOBAL#cpp_compiler_version=/usr/bin/c++ version 7.3.1 20180307 [gcc-7-branch revision 258314] (SUSE Linux) 
  NC_GLOBAL#creator_email=EOSupport@Copernicus.esa.int
  NC_GLOBAL#creator_name=The Sentinel 5 Precursor TROPOMI Level 2 products are developed with funding from the European Space Agency (ESA), the Netherlands Space Office (NSO), the Belgian Science Policy Office, the German Aerospace Center (DLR) and the Bayerisches Staatsministerium für Wirtschaft und Medien, Energie und Technologie (StMWi).
  NC_GLOBAL#creator_url=https://atmos.eoc.dlr.de/tropomi ; https://sentinel.esa.int/web/sentinel/missions/sentinel-5p
  NC_GLOBAL#date_created=2022-08-17T13:00:09Z
  NC_GLOBAL#exe_linker_flags=-static-libgcc -static-libstdc++ -static-libgfortran
  NC_GLOBAL#f90_compiler_flags=-frecursive -Wno-aggressive-loop-optimizations -O2 -fopenmp 
  NC_GLOBAL#f90_compiler_version=/usr/bin/gfortran version 7.3.1 20180307 [gcc-7-branch revision 258314] (SUSE Linux) 
  NC_GLOBAL#geolocation_grid_from_band=3
  NC_GLOBAL#geospatial_lat_max=-62.255409
  NC_GLOBAL#geospatial_lat_min=-87.085831
  NC_GLOBAL#geospatial_lon_max=107.46681
  NC_GLOBAL#geospatial_lon_min=-35.251526
  NC_GLOBAL#history=2022-08-17 13:00:09.152062 f_s5pops upas-l2 JobOrder.648198357.xml 
  NC_GLOBAL#id=S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940
  NC_GLOBAL#identifier_product_doi=N/A
  NC_GLOBAL#identifier_product_doi_authority=http://dx.doi.org/
  NC_GLOBAL#institution=DLR-IMF
  NC_GLOBAL#keywords=0325 Evolution of the atmosphere; 0340 Middle atmosphere, composition and chemistry; 0365 Troposphere, composition and chemistry; 1610 Atmosphere; 1630 Impacts of global change; 1640 Remote sensing
  NC_GLOBAL#keywords_vocabulary=AGU index terms, http://publications.agu.org/author-resource-center/index-terms/
  NC_GLOBAL#license=No conditions apply
  NC_GLOBAL#naming_authority=DLR-IMF
  NC_GLOBAL#orbit=25099
  NC_GLOBAL#platform=S5P
  NC_GLOBAL#processing_status=Nominal
  NC_GLOBAL#processor_version=02.04.01
  NC_GLOBAL#product_version=2.1
  NC_GLOBAL#project=Sentinel 5 precursor/TROPOMI
  NC_GLOBAL#references=TBD
  NC_GLOBAL#revision_control_identifier=4435
  NC_GLOBAL#sensor=TROPOMI
  NC_GLOBAL#source=Sentinel 5 precursor, TROPOMI, space-borne remote sensing, L2
  NC_GLOBAL#spatial_resolution=5.5x3.5km2
  NC_GLOBAL#standard_name_vocabulary=NetCDF Climate and Forecast Metadata Conventions Standard Name Table (v29, 08 July 2015), http://cfconventions.org/standard-names.html
  NC_GLOBAL#Status_BG=Nominal
  NC_GLOBAL#Status_L2__CLOUD_=External
  NC_GLOBAL#Status_MET_2D=Nominal
  NC_GLOBAL#Status_NISE__=Nominal
  NC_GLOBAL#Status_reference_spectrum=solar
  NC_GLOBAL#summary=TROPOMI/S5P L2 data Swath 5.5x3.5km2 processed in NRTI mode
  NC_GLOBAL#time_coverage_duration=PT260.000S
  NC_GLOBAL#time_coverage_end=2022-08-17T11:57:24Z
  NC_GLOBAL#time_coverage_resolution=PT0.840S
  NC_GLOBAL#time_coverage_start=2022-08-17T11:53:04Z
  NC_GLOBAL#time_reference=2022-08-17T00:00:00Z
  NC_GLOBAL#time_reference_days_since_1950=26526
  NC_GLOBAL#time_reference_julian_day=2459808.5
  NC_GLOBAL#time_reference_seconds_since_1970=1660694400
  NC_GLOBAL#title=TROPOMI/S5P Ozone Total Column
  NC_GLOBAL#tracking_id=c62b0f0d-1711-4850-ae91-8247baae131e
Subdatasets:
  SUBDATASET_1_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/latitude
  SUBDATASET_1_DESC=[1x302x450] latitude (32-bit floating-point)
  SUBDATASET_2_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/longitude
  SUBDATASET_2_DESC=[1x302x450] longitude (32-bit floating-point)
  SUBDATASET_3_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/delta_time
  SUBDATASET_3_DESC=[1x302x450] /PRODUCT/delta_time (32-bit integer)
  SUBDATASET_4_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/time_utc
  SUBDATASET_4_DESC=[1x302] /PRODUCT/time_utc ()
  SUBDATASET_5_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/qa_value
  SUBDATASET_5_DESC=[1x302x450] /PRODUCT/qa_value (8-bit unsigned integer)
  SUBDATASET_6_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/ozone_total_vertical_column
  SUBDATASET_6_DESC=[1x302x450] atmosphere_mole_content_of_ozone (32-bit floating-point)
  SUBDATASET_7_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/ozone_total_vertical_column_precision
  SUBDATASET_7_DESC=[1x302x450] atmosphere_mole_content_of_ozone error (32-bit floating-point)
  SUBDATASET_8_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/surface_altitude
  SUBDATASET_8_DESC=[1x302x450] surface_altitude (32-bit floating-point)
  SUBDATASET_9_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/surface_altitude_precision
  SUBDATASET_9_DESC=[1x302x450] surface_altitude standard_error (32-bit floating-point)
  SUBDATASET_10_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/surface_classification
  SUBDATASET_10_DESC=[1x302x450] /PRODUCT/SUPPORT_DATA/INPUT_DATA/surface_classification (8-bit unsigned integer)
  SUBDATASET_11_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/instrument_configuration_identifier
  SUBDATASET_11_DESC=[1x302] /PRODUCT/SUPPORT_DATA/INPUT_DATA/instrument_configuration_identifier (32-bit integer)
  SUBDATASET_12_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/instrument_configuration_version
  SUBDATASET_12_DESC=[1x302] /PRODUCT/SUPPORT_DATA/INPUT_DATA/instrument_configuration_version (16-bit integer)
  SUBDATASET_13_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/scaled_small_pixel_variance
  SUBDATASET_13_DESC=[1x302x450] /PRODUCT/SUPPORT_DATA/INPUT_DATA/scaled_small_pixel_variance (32-bit floating-point)
  SUBDATASET_14_NAME=NETCDF:"/eodata/Sentinel-5P/TROPOMI/L2__O3____/2022/08/17/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940/S5P_NRTI_L2__O3_____20220817T115218_20220817T115718_25099_03_020401_20220817T125940.nc":/PRODUCT/SUPPORT_DATA/INPUT_DATA/surface_pressure

```

**Comment by Pratichhya** [comment]
@jdries, thank you for the example. I will try this method and verify the details again. Thank you again.

**Comment by Pratichhya** [comment]
@jdries All the layers have been added as suggested. (https://eu-cdse.github.io/documentation/pr-preview/pr-216/Data/Sentinel5P.html)

Also, to resolve the merge conflict, a pull from Publish branch was done.

File: APIs/OpenSearch.qmd [hunk 1]
```diff
@@ -137,7 +137,7 @@ This will sort the output by observation date. The following orderings can be im
 
 :::
 
-Each of these ordering can be be accompanied by:
+Eeach of these ordering can be be accompanied by:
```
Comment (soxofaan) [inline APIs/OpenSearch.qmd:140]:
```suggestion
Each of these ordering can be be accompanied by:
```

File: APIs/OpenSearch.qmd [hunk 2]
```diff
@@ -298,7 +298,7 @@ To be meaningful, the cloudiness must be provided with each product, while in ma
 
 Some additional parameters are strictly satellite-specific, e.g. polarisation, which is defined only for Sentinel-1.
 
-For every satellite (collection) its set of query-able parameters may be obtained by a query such as:
+For every satellite (collection) its set of query-able parameters may be obtained by a query like:
```
Comment (soxofaan) [inline APIs/OpenSearch.qmd:301]: this looks very weird, I guess this is easier:

```suggestion
The set of query-able parameters of a given satellite (collection) can be obtained with:
```

File: APIs/S3.md [hunk 3]
```diff
@@ -12,9 +12,7 @@ To generate the necessary credentials you must have a registered account on data
 
 ## Request for secrets
 
-In order to obtain secrets,<a href= "https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new" target="_blank">Submit a request</a> and provide the e-mail address of the registered Copernicus Data Space Ecosystem user for whom the credentials will be generated.
-
-
+In order to obtain secrets, create a ticket to CDSE support. Our operators will provide you with the appropriate credentials. You must provide in such request the e-mail address of the registered CDSE user for whom the credentials will be generated.
```
Comment (soxofaan) [inline APIs/S3.md:15]: why is the link removed? this seems like a useful pointer

File: APIs/SentinelHub/Overview/ProcessingUnit.qmd [hunk 4]
```diff
@@ -80,7 +80,7 @@ apply with the following exceptions:
     factor of 1/3. Thus, three times more data can be processed
     comparing to process API for the same amount of PUs.
 -   \*\* When data is delivered to a bucket in other region within the
-    same system (i.e. Copernicus Data Space Ecosystem, AWS) there is additional cost of 0.03 PU per
+    same system (i.e. CDAS, AWS) there is additional cost of 0.03 PU per
```
Comment (soxofaan) [inline APIs/SentinelHub/Overview/ProcessingUnit.qmd:83]: CDAS? is that an intended change?

File: APIs/SentinelHub/UserGuides/Transparency.qmd [hunk 5]
```diff
@@ -27,7 +27,7 @@ that the JPEG format does not support alpha channel.
 ### Transparent no-data pixels
 
 No-data pixels are marked by the value 0 in the `dataMask` band. In the
-evalscript below, we return value 0 in the fourth band for such pixels,
+evalscript below, we return value 0 in the forth band for such pixels,
```
Comment (soxofaan) [inline APIs/SentinelHub/UserGuides/Transparency.qmd:30]: forth = onward
fourth = 4th

I guess the latter is meant, no?

File: APIs/openEO/Python_Client/Python.qmd [hunk 6]
```diff
@@ -27,7 +27,7 @@ The openEO Python client library is available on [PyPI](https://pypi.org/project
 and can easily be installed with a tool like `pip`, for example:
 
 ```bash
-pip install openeo
+pip instal openeo
```
Comment (soxofaan) [inline APIs/openEO/Python_Client/Python.qmd:30]:
```suggestion
pip install openeo
```

File: APIs/openEO/R_Client/R.qmd [hunk 7]
```diff
@@ -115,7 +115,7 @@ since we just queried publicly available back-end information.
 However, to run non-trivial processing queries one has to authenticate
 so that permissions, resource usage, etc. can be managed properly.
 
-To authenticate your account on the backend of the Copernicus Data Space Ecosystem, it is necessary for you to complete the registration process. Once registered,  the OIDC ([OpenID Connect](https://openid.net/connect/){target="_blank"}) authentication method will be employed to verify your identity using an external service.
+To authenticate your account on the backend of the Copernicus Data Space Ecosystem, it is necessary for you to complete the registration process. Once registered,  the OIDC ([OpenID Connect](https://openid.net/connect/){target="_blank"})authentication method will be employed to verify your identity using an external service.
```
Comment (soxofaan) [inline APIs/openEO/R_Client/R.qmd:118]:
```suggestion
To authenticate your account on the backend of the Copernicus Data Space Ecosystem, it is necessary for you to complete the registration process. Once registered,  the OIDC ([OpenID Connect](https://openid.net/connect/){target="_blank"}) authentication method will be employed to verify your identity using an external service.
```

File: FAQ.qmd [hunk 8]
```diff
@@ -178,7 +178,7 @@ title : "Frequently Asked Questions"
 <details>
   <summary>How can I request to change my Copernicus user type from Copernicus General user to Copernicus Services user?  
   </summary>
-    <p>To change your Copernicus user type, submit a request to our support team via <a href= "https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new" target="_blank">Submit a request</a>.  Please select 'Other' for the ‘I have a question about*:’ and provide the following information: 
+    <p>To change your Copernicus user type, submit a request with our support team via <a href= "https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new" target="_blank">Submit a request</a>.  Please select 'Other' for the ‘I have a question about*:’ and provide the following information:
```
Comment (soxofaan) [inline FAQ.qmd:181]: ?

File: Registration.qmd [hunk 9]
```diff
@@ -36,7 +36,7 @@ Upon registration, you will be prompted to verify your email address by receivin
 
 After successfully verifying your email, your registration process is complete. You can now log in using your credentials (providing Email and Password).
 
-If you have an issue with registering or you want to deregister, please [contact us](mailto://help-login@dataspace.copernicus.eu?Subject=Subject%20Text&Body=Your%20comments).
+If you have an issue with registering or you want to deregister, please [contact us](mailto://help-login@dataspace.copernicus.eu?Subject=Subject%20Text&Body=Your%20comments) directly.
```
Comment (soxofaan) [inline Registration.qmd:39]: > ?Subject=Subject%20Text&Body=Your%20comments

It makes no sense to me to use these `subject` and `body` parameters if you are just putting generic placeholders anyway

File: Roadmap/AppTable.qmd [hunk 10]
```diff
@@ -23,7 +23,7 @@ format:
         <tr>
             <td rowspan="1" bgcolor="#f7f8fa">Identity service</td>
             <td>User Registration and identity management</td>
-            <td>Start of registration of users allowing to access first free services on the Copernicus Data Space Ecosystem. Management of user and organization.</td>
+            <td>Start of registration of users allowing to access first free services on CDE. Management of user and organization.</td>
```
Comment (soxofaan) [inline Roadmap/AppTable.qmd:26]: CDSE?

**Review by soxofaan — COMMENTED** [review]
This PR feels very sloppy as it introduces typos in quite some places

**Comment by Pratichhya** [comment]
Hi @jdries and @soxofaan ,
 Thank you for your time and effort in reviewing this PR. But it seems like it's now a bit mixed and messed up 😓 caused when trying to resolve the merging conflict.

There, I think it's good to close this and create a fresh one for a dedicated issue. 

Also, @soxofaan, thank you for noticing and notifying such mistakes, I will try to update the typo in a [separate PR](https://github.com/eu-cdse/documentation/issues/396) and request CF for those that need their suggestions.

File: APIs/S3.md (hunk 3 above)
Comment (Pratichhya) [reply APIs/S3.md:15]: not sure what was pulled to avoid conflict

File: APIs/SentinelHub/Overview/ProcessingUnit.qmd (hunk 4 above)
Comment (Pratichhya) [reply APIs/SentinelHub/Overview/ProcessingUnit.qmd:83]: No it was not intended
It seems like the old changes were accepted by mistake that cause acceptance of old issues. However, updated information are already existing in the current production version

**Pull request closed without merging**